webdriver-spec.html

```diff
@@ -2995,12 +2995,19 @@
  run the following steps:
 
 <ol>
+ <li><p>Start a <var>timer</var>. If this algorithm has not
+  completed before <var>timer</var> reaches
+  the <a>session</a>'s <a>session page load timeout</a> in
+  milliseconds, return an <a>error</a> with <a>error
+  code</a> <a>timeout</a>.
```

Comment:
If the script timeout is set to null, it will evaluate the injected script forever or until it resolves/returns. Should we allow the same for the page load duration?